selfdrive/updated.py

```diff
@@ -49,6 +50,8 @@
 NICE_LOW_PRIORITY = ["nice", "-n", "19"]
 SHORT = os.getenv("SHORT") is not None
```

Comment:
SHORT is no longer used?

Reply:
It's from way back, before the version of the updater I built in December 2019 with the SIGHUP handler. I'll garbage-collect it. Short-cycling the updater can potentially cause strange behavior I don't want to bring it back.